channel/funder.go

```diff
@@ -401,6 +427,81 @@ loop:
 	return nil
 }
 
+// EgoisticWaitForFundingConfirmation waits for the confirmation events on the blockchain that
+// all peers except oneself (request.Idx) successfully funded the channel for the specified asset
+// according to the funding agreement.
+func (f *Funder) EgoisticWaitForFundingConfirmation(ctx context.Context, request channel.FundingReq, asset assetHolder, fundingIDs [][32]byte) error {
```

Comment:
As @sophia1ch pointed out, this method only waits for the other participants to fund a specific asset which is not desired since we rather want to wait until all assets are funded before proceeding with our own funding.